charts/jupyter-python/values.schema.json

```diff
@@ -543,6 +543,14 @@
             "form": true,
             "title": "Ingress Details",
             "properties": {
+                "enabled": {
```

Comment:
i think we should put a `default : true` because if an onyxia instance do not define ingress or route ( for example an old version of onyxia ) we should assume that ingress is enabled and route disabled

Reply:
done